Cargo.toml

```diff
@@ -12,15 +12,14 @@ description = """
     use of tendermint-rs.
 """
 
-[[bin]]
-name = "basecoin"
+default-run = "basecoin"
```

Comment:
I was thinking we could have a trial run for `basecoin-tower`, in case a lot of things break with it. Maybe we should run it as default in the CI for some time?

Reply:
> I also was thinking, wouldn't be better to keep one of the tendermint-abci or tower-abci as a feature, allowing us to run both in CI and without to have duplicating codes and maintaining two binaries? This way, we also can keep tendermint-abci around longer in case some testing scenarios need it

We can do so. What about feature gating? wouldn't it be better to include both?

Reply:
Actually, I was not looking for reducing the binary size. Something I just realized and missed here... the tower-abci `main()` function is async, and runs under the `#[tokio::main]`. So, it's not possible to go with only one binary by feature gating. Right.